src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -95,12 +95,12 @@ export class QueryEnhancementsPlugin
     queryString.getLanguageService().registerLanguage(sqlLanguageConfig);
 
     data.__enhance({
```

Comment:
nit: Can we change the `__enhance` name in the API and directly call the language service API's. We are exposing the language service right?